sdcm/nemesis.py

```diff
@@ -3782,6 +3799,7 @@ def disrupt(self):
 class NodeRestartWithResharding(Nemesis):
     disruptive = True
     kubernetes = True
+    topology_changes = True
```

Comment:
I'm not sure it's considered as a topology change, but I'm ok to leave it to be on the safe side.